src/webextension/icons/chevron-right.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's run this through `svgo`; as mentioned elsewhere, whatever the settings @changecourse used for the other SVGs should be fine.

Reply:
>Let's run this through svgo; as mentioned elsewhere, whatever the settings @changecourse used for the other SVGs should be fine.

@jimporter Nice catch 🎖  

@nickbrandt Can you please update the svg here for consistency before we merge?